apps/discovery_api/test/utils/test_helper.ex

```diff
@@ -92,7 +92,7 @@ defmodule DiscoveryApi.Test.Helper do
   end
 
   def wait_for_brook_to_be_ready() do
-    Process.sleep(5_000)
+    Process.sleep(20_000)
```

Comment:
ideally this function could check something in brook to see if it's happy yet

Reply:
💯 If this works, that would be the ideal scenario.